liveness/src/main/java/com/amplifyframework/ui/liveness/ui/FaceLivenessDetector.kt

```diff
@@ -386,3 +401,21 @@ internal fun ChallengeView(
         }
     }
 }
+
+private fun isFaceMovementAndLightChallenge(livenessSessionInfo: FaceLivenessSession?): Boolean =
```

Comment:
Looks like a good extension function.
```suggestion
private fun FaceLivenessSession?.isFaceMovementAndLightChallenge(): Boolean =
```